packages/doc/content/components/helpers/media.mdx

```diff
@@ -0,0 +1,138 @@
+---
+title: 'Media'
+metaTitle: 'Media'
+metaDescription: 'Functions to help you dealing with responsive design'
+---
+
+import { Link } from 'gatsby';
+
+## Media
+
+The `media` module makes easy to write media queries using `styled-components`.
+
+It has a helper for each breakpoint, defined in our <Link to="/guidelines/tokens/breakpoints">guidelines</Link>:
+`xxs`, `xs`, `sm`, `md`, `lg`, `xl`, `xxl`, and `xxxl`.
+
+### Installing
+
+```bash type=highlight
+yarn add @gympass/yoga-helpers
+
+// or
+
+npm install @gympass/yoga-helpers
+```
+
+### Usage
+
+```javascript type=highlight
+import styled from 'styled-components';
+import { media } from '@gympass/yoga-helper';
+​
+const MyResponsiveButton = styled.button`
+  // default min
+  // @media (min-width: breakpoint)
+  ${media.md``}
+​
+  // you can also use as media.min(breakpoint)
+  // @media (min-width: breakpoint)
+  ${media.min('md')``}
+
+  // or media.max('breakpoint')
+  // @media (max-width: breakpoint)
+  ${media.max('xl')``}
+​
+  // for a device range
+  // @media (min-width: breakpoint) and (max-width: breakpoint)
+  ${media.between('md', 'l')``}
+​
+  // jest like
+  // @media not all and (min-width: breakpoint)
+  ${media.not.md``}
+
```

Comment:
what do you think about including this `media.not.min`?

Reply:
Could you review it again, please?